ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -63,8 +80,22 @@ module.exports = Marionette.LayoutView.extend({
   },
   serializeData: function() {
     let modelJSON = this.model.toJSON()
+    let summaryModel = {}
+    const summary = props.summaryShow
+    summary.forEach(attribute => {
+      if (this.model.get('targetMetacard') !== undefined)
```

Comment:
why is this only hardcoded for target metacards?

Reply:
TargetMetacard` is an overloaded term. In this case, `targetMetacard` refers to the metacard being selected on the UI. This was tested by using non-target imports and this code still grabs the correct metacard even though it is not a targetMetacard.